flask_app.py

```diff
@@ -232,22 +254,75 @@ def check_identity_or_auth(request):
     if access_token is None:
         log.debug(f"Incoming request with no access token, telling them to auth the app")
         link = make_authorization_url()
-        resp = make_response(render_template('auth.html', authlink=link))
+        discordlink = make_discord_authorization_url()
+        resp = make_response(render_template('auth.html', authlink=link, discordauthlink=discordlink))
```

Comment:
Probably best to update names like `authlink` and `make_authorization_url()` to `redditauthlink` and `make_reddit_authorization_url()` now that we have 2.